avm/res/web/site/README.md

```diff
@@ -2123,6 +2197,7 @@ module site 'br/public:avm/res/web/site:<version>' = {
 | [`privateEndpoints`](#parameter-privateendpoints) | array | Configuration details for private endpoints. For security reasons, it is recommended to use private endpoints whenever possible. |
 | [`publicNetworkAccess`](#parameter-publicnetworkaccess) | string | Whether or not public network access is allowed for this resource. For security reasons it should be disabled. If not specified, it will be disabled by default if private endpoints are set. |
 | [`redundancyMode`](#parameter-redundancymode) | string | Site redundancy mode. |
+| [`retainExistingSettings`](#parameter-retainexistingsettings) | bool | Retain existing app settings. If set to true, existing app settings which are NOT defined in the Bicep file will be retained. Settings which are defined in the Bicep file will be updated irrespective of this parameter. |
```

Comment:
Instead adding this conditionally, I recommend we do not add any new option, but module should by default take care of not removing any config, since bicep by default is stateless

Reply:
It's a design decision at the end, and would be possible. In that case this change would be a breaking change (default behavior has changed), and then the minor version number should be increased.

Reply:
i think thats ok to do a minor version bump, because that is the expected behavior and instead of adding new parameter to control this, we should have it by default. No one would like to loose the config that is already present. 

what do you think @tsc-buddy ?

Reply:
@peterbud Would you like to make the change so we can move forward with this ?

Reply:
I assumed you were waiting on @tsc-buddy 's response.

Sure I can update the PR

Reply:
Lets do it.

Reply:
Done

Reply:
@peterbud Last Workflow runs seems to be failing.

Reply:
My subscription is rather limited, depending on which region is picked, some of the jobs are failing. If you rerun they succeed, se here:
[![avm.res.web.site](https://github.com/peterbud/bicep-registry-modules/actions/workflows/avm.res.web.site.yml/badge.svg)](https://github.com/peterbud/bicep-registry-modules/actions/workflows/avm.res.web.site.yml)

Reply:
Yeah its a problem